crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -2,62 +2,82 @@
 <!DOCTYPE html>
 <html>
  <head>
-   <meta charset="utf-8">
```

Comment:
Note ~ many of the diffs in this file are caused by me bringing `base.html` into a consistent two-spaces-for-tab format; on `master` a mixture of different spaces and tabs are used.